### ISSUE TITLE
Does it work with Azure openai? 

### ISSUE THREAD

**Issue by 4722794** [livekit-examples/realtime-playground#6]
Great project! 

Is it possible to make it work with Azure openai since the services and offerings are identical?

**Comment by mapmakerrg** [comment]
Yes we need it to work with AzureOpenAI as well .. please please

**Comment by mapmakerrg** [comment]
who can help out ?

**Comment by illgitthat** [comment]
It looks like it is being worked on here: https://github.com/livekit/agents/pull/848
